CRM/Extension/Downloader.php

```diff
@@ -75,11 +95,9 @@ public function checkRequirements($extensionInfo = NULL) {
       $requiredExtensions = CRM_Extension_System::singleton()->getManager()->findInstallRequirements([$extensionInfo->key], $extensionInfo);
       foreach ($requiredExtensions as $extension) {
         if (CRM_Extension_System::singleton()->getManager()->getStatus($extension) !== CRM_Extension_Manager::STATUS_INSTALLED && $extension !== $extensionInfo->key) {
-          $requiredExtensionInfo = CRM_Extension_System::singleton()->getBrowser()->getExtension($extension);
-          $requiredExtensionInfoName = empty($requiredExtensionInfo->name) ? $extension : $requiredExtensionInfo->name;
           $errors[] = [
             'title' => ts('Missing Requirement: %1', [1 => $extension]),
-            'message' => ts('You will not be able to install/upgrade %1 until you have installed the %2 extension.', [1 => $extensionInfo->name, 2 => $requiredExtensionInfoName]),
+            'message' => ts('You will not be able to install/upgrade %1 until you have installed the %2 extension.', [1 => $extensionInfo->name, 2 => $extension]),
```

Comment:
This undoes https://github.com/civicrm/civicrm-core/pull/21852 - just wondering if it's on purpose or a rebase/conflict resolution mixup?

Reply:
@demeritcowboy Gone! thanks that was a rebase/mixup